srcpkgs/python3-python-multipart/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'python3-python-multipart'
+pkgname=python3-python-multipart
+version=0.0.9
+revision=1
+build_style=python3-pep517
+hostmakedepends="hatchling"
+checkdepends="python3-pytest python3-yaml"
```

Comment:
should yaml also be in depends?

Reply:
```
$ grep -r yaml                                                                                                                     master 
tests/test_multipart.py:import yaml
tests/test_multipart.py:        yaml_file = os.path.join(http_tests_dir, fname + ".yaml")
tests/test_multipart.py:        with open(yaml_file, "rb") as f:
tests/test_multipart.py:            yaml_data = yaml.safe_load(f)
tests/test_multipart.py:        http_tests.append({"name": fname, "test": test_data, "result": yaml_data})
CHANGELOG.md:* Use yaml.safe_load instead of yaml.load [#46](https://github.com/andrew-d/python-multipart/pull/46).
```

Nope, just for test :confused: